docs/UserGuide.md

```diff
@@ -83,46 +84,54 @@ Examples:
 * `client add 5 n/Katya p/98123456 e/katya@yahoo.com a/Vladivostok, Nevelskogo, bld. 15, appt. 256 c/RUS tz/GMT+3`
 <br> This adds a new client with name **Katya**, phone number **98123456**, email **katya@yahoo.com**, address **Vladivostok,
 Nevelskogo, bld. 15, appt. 256**, country **Russia**, timezone **GMT+3**.
-### Listing all persons : `list`
 
-Shows a list of all persons in the address book.
+### Viewing clients: `client view`
 
-Format: `list`
+View the client specified by the index. If no index is specified, all clients will be listed.
 
-### Editing a person : `edit`
+Format: `client view [INDEX]`
 
-Edits an existing person in the address book.
+Examples:
+* `client view 2` Views information for client at index 2
+* `client view` Lists all clients
 
-Format: `edit INDEX [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [t/TAG]…​`
+### Finding clients: `client find`
 
-* Edits the person at the specified `INDEX`. The index refers to the index number shown in the displayed person list. The index **must be a positive integer** 1, 2, 3, …​
-* At least one of the optional fields must be provided.
-* Existing values will be updated to the input values.
-* When editing tags, the existing tags of the person will be removed i.e adding of tags is not cumulative.
-* You can remove all the person’s tags by typing `t/` without
-    specifying any tags after it.
+Searches for clients that matches the given name.
```

Comment:
Might want to change this to match the original "Finds clients whose names contain any of the given keywords.", and change `n/NAME` to `KEYWORD [MORE_KEYWORDS]` as the format section below mentions keywords

Reply:
alright, originally copied from our hackmd document, which seems to be out of date based on what you pointed out
![image](https://user-images.githubusercontent.com/38996397/94329696-ead02e80-ffef-11ea-80f1-7340f64f1d03.png)